nixos/tests/ipv6.nix

```diff
@@ -86,6 +87,14 @@ import ./make-test-python.nix ({ pkgs, lib, ...} : {
           # Default route should have "src <temporary address>" in it
           client.succeed(f"ip r g ::2 | grep {ip}")
 
+      with subtest("disabling privacy extensions by default"):
```

Comment:
I wasn't aware we had IPv6 tests: there is some duplication between `nixos/tests/ipv6.nix` and `nixos/tests/networking.nix`. I wrote a test for temporary addresses there that we should merge with this, maybe.

Reply:
While I agree that the duplication should be removed, it's a task I'd rather relegate to a separate PR.